.github/workflows/optimize-ci.yml

```diff
@@ -341,92 +341,13 @@ jobs:
         with:
           archive_name: integration-tests-opensearch-${{ matrix.includedGroups }}-docker
 
-  integration-tests-os-dev:
```

Comment:
This pipeline makes no sense in C8, so removing it

Reply:
What was the context of it? Is it also removable for Optimzie 7?